app/src/context/store.tsx

```diff
@@ -71,6 +72,8 @@ const {Provider} = store;
 
 const StateProvider = (props: any) => {
   const [initialized, setInitialized] = useState(false);
+  const startedInitialisation = useRef(false);
```

Comment:
@stevecassidy  this records whether we have started the initialisation instead of relying on global let variable - more idiomatic for react.